src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -360,13 +360,27 @@ private void TabViewItem_Loaded(object sender, RoutedEventArgs e)
 
 		private void DragAreaRectangle_Loaded(object sender, RoutedEventArgs e)
 		{
-			double scaleAdjustment = DragAreaRectangle.XamlRoot.RasterizationScale;
-			double titleBarInset = ((FilePropertiesHelpers.FlowDirectionSettingIsRightToLeft
-				? MainWindow.Instance.AppWindow.TitleBar.LeftInset
-				: MainWindow.Instance.AppWindow.TitleBar.RightInset) / scaleAdjustment) + 40;
+			if (HorizontalTabView.ActualWidth > 0 && TabBarAddNewTabButton.Width > 0)
```

Comment:
Isn't HorizontalTabView already loaded at this point?

Reply:
The downside to this logic is that it can result in a loop if an issue causes the value to remain 0. I suggest adding a short delay to accommodate slower devices and, if the value is still 0, we can use a fallback value for that device. However, I expect a short delay to be more than adequate.

Reply:
The solution may be to add only one repeat, since it is queued at the end and called when everything is ready.

Reply:
That's better, however I still prefer the logic of adding a short delay.

Reply:
If I add a delay, the problem is solved on slow machines, but the delay may be visible on other machines. This workaround will take care of the fast machines, and the slow machines will have a delay due to queuing, but at the same time the queuing will only be done once. "Personally, I try to avoid Delay when programming."

Reply:
I mean like this
```cs
private async void DragAreaRectangle_Loaded(object sender, RoutedEventArgs e)
{
    if (HorizontalTabView.ActualWidth <= 0 || TabBarAddNewTabButton.Width <= 0)
        await Task.Delay(100);

    var appWindow = MainWindow.Instance.AppWindow;
    var titleBarInset = (FilePropertiesHelpers.FlowDirectionSettingIsRightToLeft
        ? appWindow.TitleBar.LeftInset
        : appWindow.TitleBar.RightInset) / DragAreaRectangle.XamlRoot.RasterizationScale;

    if (titleBarInset > 0)
    {
        titleBarInset += 40; // Add 40px gap
        RightPaddingColumn.Width = new(titleBarInset);

        HorizontalTabView.Measure(new(
            HorizontalTabView.ActualWidth - TabBarAddNewTabButton.Width - titleBarInset,
            HorizontalTabView.ActualHeight));
    }
}
```

Reply:
OK, sorry, it wasn't in my mind, sure, maybe I'm too invasive in my thoughts. 😅